cves/kernel/CVE-2013-3224.yml

```diff
@@ -280,8 +301,8 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: There was no IPC involved in this case; it was majorly a flaw in the implementation of the bluetooth protocol.
```

Comment:
Is bluetooth not a form of IPC?

Reply:
Responding to AitanBachrach - maybe, but the vulnerability itself does not seem to have anything to do with IPC

Reply:
Yeah I wasn't really sure, but after rereading the question, I guess the vulnerability did affect the feature which used IPC. In this case, the bluetooth protocol which was using sockets to deal with data. Although that is the case, the vulnerability itself did not really have anything to do with IPC.  I have updated it with a summary of both answers.